middleware/markdown/markdown_tpl.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>{{.Title}}</title>
+</head>
+<body>
+{{.Include "header.html"}}
+Welcome to {{.Var.sitename}}!
```

Comment:
This PR is looking really good. Is it possible to combine Var, Title, and Markdown into a single map, maybe, so that all values can still be accessed by .title, .markdown, and .sitename without having to impose .Var before user variables?